src/Uno.UI/UI/Xaml/Controls/NavigationView/NavigationBackButton.xaml

```diff
@@ -14,8 +19,9 @@
         <Setter Property="HorizontalContentAlignment" Value="Center" />
         <Setter Property="VerticalContentAlignment" Value="Center" />
         <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
-        <Setter Property="Content" Value="&#xE72B;" />
-        <Setter Property="Template">
+		<win:Setter Property="Content" Value="&#xE72B;" />
+		<xamarin:Setter Property="Content" Value="&#xE112;" />
```

Comment:
I would be great to remove this difference. Maybe we should fix the TTF directly?